tests/queries/0_stateless/03321_initial_query_start_time_check.sql

```diff
@@ -0,0 +1,10 @@
+-- Tags: no-fasttest, no-replicated-database
+-- no-fasttest: Timeouts are slow
+
+DROP TABLE IF EXISTS tmp;
+
+CREATE OR REPLACE VIEW tmp AS SELECT initialQueryStartTime() as it, now() AS t FROM system.one WHERE NOT ignore(sleep(0.1));
```

Comment:
If I am right, it should finish in ~2 seconds? If so, it's okay to add it to the fasttests

Reply:
Yes, it should complete within 2 seconds. Alright, I’ll add it to the fast testtests.

Reply:
while it is launched in parallel, it's fine to keep it